src/theme/theme.js

```diff
@@ -0,0 +1,29 @@
+import { createMuiTheme } from '@material-ui/core/styles'
+import amber from '@material-ui/core/colors/amber'
+import grey from '@material-ui/core/colors/grey'
+import indigo from '@material-ui/core/colors/indigo'
+
+const theme = createMuiTheme({
+  palette: {
+    // type: 'dark',
+    primary: {
+      main: amber[500],
+    },
+    secondary: {
+      main: grey[50],
```

Comment:
make sure to use the colors on your mockup instead.
declare the  colors as variables and import and use them here.